vocabularies/Common.xml

```diff
@@ -795,8 +795,15 @@ The minimum structure is
         </Property>
       </ComplexType>
       <ComplexType Name="ValueListParameterIn" BaseType="Common.ValueListParameter">
+        <Property Name="Operator" Type="UI.SelectionRangeOptionType" DefaultValue="0" Nullable="false">
```

Comment:
Instead of extending the existing type, define a new subtype (suggested name `ValueListRangeIn`) of `ValueListParameterIn` and add the new properties to that subtype only.

Reply:
=> https://github.com/SAP/odata-vocabularies/commit/2e24e353bd0f15f9e82d3adc4244e098f2f2cea1